### PR TITLE
[4.0] Improve form validation styling

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

This PR uses some of the classes from the BS4 form validation: https://v4-alpha.getbootstrap.com/components/forms/#examples

### CODE REVIEW

File: media/system/js/fields/validate.js [hunk 1]
```diff
@@ -110,18 +114,19 @@ var JFormValidator = function() {
...
-		    el.classList.remove('valid');
+			el.classList.remove('form-control-danger');
+			el.classList.remove('form-control-success');
+			el.parentNode.classList.remove('has-danger');
+			el.parentNode.classList.remove('has-success');
 
 		    // Remove message
 		    if (message) {
 			    if (el.parentNode.tagName.toLowerCase() === 'span' && el.parentNode.classList.contains('input-group')) {
-				    console.log(el.parentNode.parentNode)
```
Comment (brianteeman) [inline media/system/js/fields/validate.js]: what is the reasoning to remove the console log - i would think that was useful

Comment (brianteeman) [inline media/system/js/fields/validate.js]: ignore that - you did a new commit that answers the question as i was posting

Comment (C-Lodder) [inline media/system/js/fields/validate.js]: This isn't returning a data response, it's simply returning an element. No need

Comment (brianteeman) [inline media/system/js/fields/validate.js]: Ok

**Pull request merged**